src/app/auth/features/user-manager-list/user-manager-list.component.html

```diff
@@ -0,0 +1,109 @@
+@if (usersAreLoading()) {
+  <p class="text-base">Loading users...</p>
+} @else {
+  @let columns = ['email', 'name', 'roles', 'status', 'registration_date', 'details'];
+  <div class="overflow-auto">
+    <table
+      mat-table
+      [dataSource]="source"
+      class="mat-elevation-z8"
+      matSort
+      #sort="matSort"
+    >
+      <ng-container matColumnDef="email">
+        <th mat-header-cell *matHeaderCellDef class="min-w-48" mat-sort-header>
+          Email
+        </th>
+        <td mat-cell *matCellDef="let user">
+          {{ user.email }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="name">
+        <th mat-header-cell *matHeaderCellDef class="min-w-40" mat-sort-header>
+          User Name
+        </th>
+        <td mat-cell *matCellDef="let user">
+          {{ user.displayName }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="roles">
+        <th mat-header-cell *matHeaderCellDef class="w-40" mat-sort-header>Roles</th>
+        <td mat-cell *matCellDef="let user">
+          <span class="flex flex-wrap gap-1">
+            @for (roleName of user.roleNames; track roleName) {
+              <mat-chip>{{ roleName }}</mat-chip>
+            }
+          </span>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="status">
+        <th
+          mat-header-cell
+          *matHeaderCellDef
+          class="min-w-24 capitalize"
+          mat-sort-header
+        >
+          Status
+        </th>
+        <td
+          mat-cell
+          class="{{ user.status | UserStatusClassPipe }} capitalize"
+          *matCellDef="let user"
+        >
+          {{ user.status }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="registration_date">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header>Registration Date</th>
+        <td mat-cell *matCellDef="let user">
+          {{ user.registration_date | date }}
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="details">
+        <th mat-header-cell *matHeaderCellDef class="w-24">Details</th>
+        <td mat-cell *matCellDef="let user">
+          <button
+            mat-icon-button
+            (click)="viewDetails(user); $event.stopPropagation()"
+            aria-label="View user details"
+            class="text-primary"
+          >
+            <mat-icon>chevron_right</mat-icon>
+          </button>
+        </td>
+      </ng-container>
+
+      <tr mat-header-row *matHeaderRowDef="columns"></tr>
+      <tr
+        mat-row
+        *matRowDef="let row; columns: columns"
+        class="duration-200 hover:cursor-pointer hover:bg-black/[0.05] hover:transition"
+        (click)="openDetails(row)"
+      ></tr>
```

Comment:
We should add some highlighting on hover, much like the other management tables, but this can be in a separate PR.

Reply:
I have added a separate task GSI-1772 to make the look and feel consistent across all admin modules.